.gitlab-ci.yml

```diff
@@ -57,5 +57,41 @@ deploy-python-tool-production:
     ENVIRONMENT_NAME: production
     EVA_SUBMISSION_PATH: $EVA_SUBMISSION_PRODUCTION_PATH
     DEPLOY_POINT: $CI_COMMIT_TAG
+  only:
+    - tags
+
+# Not executed, parent job definition
+.deploy-python-tool-codon:
+  stage: deploy
+  image: python:3.8.7-slim-buster
+  environment:
+    name: $ENVIRONMENT_NAME
+  script:
+    # Start ssh-agent and set identity then run deploy script
+    # See https://docs.gitlab.com/ee/ci/ssh_keys/
+    - apt-get update
+    - apt-get -y install openssh-client
+    - eval "$(ssh-agent)"
+    - echo "$EVA_ETL_PRIVATE_KEY" | tr -d '\r' | ssh-add -
```

Comment:
Are we going to use the same key ?

Reply:
As discussed, for now we can do so as it's already been copied over.